xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamBluray.h

```diff
@@ -10,10 +10,15 @@
 
 #include "BlurayStateSerializer.h"
 #include "DVDInputStream.h"
+#include "IVideoPlayer.h"
```

Comment:
To introduce a dependency on the video player interface here is wrong, from the architectural pov.